### PR TITLE
Fix version issue with mac and win binaries

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

**Comment by sebastienvas** [comment]
Waiting for someone to test https://storage.googleapis.com/istio-artifacts/sebtest/istioctl-osx and confirm istioctl version is printing proper version.

**Comment by ldemailly** [comment]
```
$  ~/Downloads/istioctl-osx  version
Version: dc29ca7-dirty
GitRevision: dc29ca7-dirty
GitBranch: master
User: sebastienvas@sebastienvas-glaptop.roam.corp.google.com
GolangVersion: go1.8.3
```

File: bin/upload-istioctl [hunk 1]
```diff
@@ -43,7 +43,7 @@ if [[ ${RELEASE} == true ]]; then
   LDFLAGS=""
   while read line; do
     read SYMBOL VALUE < <(echo $line)
-    LDFLAGS=${LDFLAGS}" -X istio.io/pilot/cmd/version.${SYMBOL}=${VALUE}"
+    LDFLAGS=${LDFLAGS}" -X istio.io/pilot/tools/version.${SYMBOL}=${VALUE}"
```
Comment (ldemailly) [inline bin/upload-istioctl]: so before this was error'ing out but being ignored ?

Comment (sebastienvas) [inline bin/upload-istioctl]: I think it was silently failing. No error printed no warning.

**Comment by ldemailly** [comment]
thx a lot for a very fast fix ! greatly appreciated!

**Pull request merged**